bamboo/docker-compose-local.yml

```diff
@@ -23,7 +23,8 @@ services:
       - 8080:8080
       - 9200:9200
   localstack:
-    image: localstack/localstack:0.11.5
+    # image: localstack/localstack:0.11.5
+    image: localstack/localstack:0.12.13
```

Comment:
This is necessary in order for the `deleteObjects` SDK call for S3 (used in `packages/api/lambdas/process-s3-dead-letter-archive.js`) to be supported in LocalStack